src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -3,16 +3,16 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>4.0.1</Version>
+    <Version>4.0.2</Version>
```

Comment:
Should we be pushing alpha version for test before the final version is pushed here?